.github/workflows/ci.yml

```diff
@@ -95,25 +121,33 @@ jobs:
   #ubuntu 22.04 defaults to openssl3 version by default when installing
   #libssl-dev package. Hence we can rely on OS version to pull in openssl3.
   linux-openssl3-static:
-    runs-on: ubuntu-22.04 # latest
+    runs-on: ubuntu-24.04 # latest
```

Comment:
same thing about comment

Reply:
same thing here. this is just updating what it runs on